gitops/argo-apps/linkerd-control-plane.yaml

```diff
@@ -8,7 +8,7 @@ spec:
   source:
     chart: linkerd-control-plane
     repoURL: https://helm.linkerd.io/stable
-    targetRevision: 2.12.0
+    targetRevision: 1.9.3
```

Comment:
This should be 1.9.0 instead, to point to stable-2.12.0. And in the gitops tutorial folks are supposed to bump to 1.9.3 in order to have argo upgrade to stable-2.12.1 :wink: